app/features/receive/cashu-receive-quote-service.ts

```diff
@@ -185,6 +186,8 @@ export class CashuReceiveQuoteService {
       mintingFee: receiveQuote.mintingFee,
     };
 
+    const { paymentHash } = decodeBolt11(receiveQuote.mintQuote.request);
```

Comment:
should we maybe put this into `CashuReceiveLightningQuote` and then set in `getLightningQuote` since we are doing sort of ln invoice data preparation there?